### PR TITLE
[ADDENDUM] HBASE-27178 Use spotless to format code (including scala c…

### PR DESCRIPTION
**Pull request by NihalJain**
…ode)

- Add config for imports rewrite
- Add config for handling newlines with curly lambda params 
- Refactor file with more comments

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 33s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  2s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
|  |   |   0m 54s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.43 ServerAPI=1.43 base: https://ci-hbase.apache.org/job/HBase-Connectors-PreCommit/job/PR-121/1/artifact/yetus-precommit-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase-connectors/pull/121 |
| Optional Tests | dupname |
| uname | Linux 227b41a5084e 5.4.0-152-generic #169-Ubuntu SMP Tue Jun 6 22:23:09 UTC 2023 x86_64 GNU/Linux |
| Build tool | hb_maven |
| Personality | dev-support/jenkins/hbase-personality.sh |
| git revision | master / c1eb13a |
| Max. process+thread count | 34 (vs. ulimit of 12500) |
| modules | C:  U:  |
| Console output | https://ci-hbase.apache.org/job/HBase-Connectors-PreCommit/job/PR-121/1/console |
| versions | git=2.20.1 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: dev-support/.scalafmt.conf [hunk 1]
```diff
@@ -16,18 +16,39 @@
...
 align.openParenDefnSite = false
 align.openParenCallSite = false
 align.tokens = []
+
+# Rewrites
+rewrite.rules = [Imports]
+
+# Imports
+rewrite.imports.sort = scalastyle
+rewrite.imports.groups = [
```
Comment (NihalJain) [inline dev-support/.scalafmt.conf:30]: This order is based on what we have for java. See https://github.com/apache/hbase-connectors/blob/master/dev-support/eclipse.importorder

**Comment by Reidddddd** [comment]
Please change the commit title, ticket number comes first
`HBASE-27178 [Addendum]*`

**Comment by NihalJain** [comment]
This change is already merged as part of https://github.com/apache/hbase-connectors/pull/120. Closing this!

**Pull request closed without merging**